src/main/java/org/schemaspy/Config.java

```diff
@@ -1374,6 +1375,32 @@ public boolean isPaginationEnabled() {
 
         return paginationEnabled;
     }
+
+
+    /**
+     * If enabled SchemaSpy will load from classpath additional jars used by JDBC Driver<p/>
+     * <p>
+     * Defaults to <code>false</code> (enabled).
+     *
+     * @param enabled
+     */
+    public void setLoadJDBCJarsEnabled(boolean enabled) {
+        loadJDBCJarsEnabled = enabled;
+    }
+
+    /**
+     * @return
+     * @see #setLoadJDBCJarsEnabled(boolean)
+     */
+    public boolean isLoadJDBCJarsEnabled() {
+        String loadJars = pullParam("-loadjars");
+        if (loadJars != null && loadJars.equals("true")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)